test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/NpgsqlCommandTests.cs

```diff
@@ -33,23 +31,13 @@ public void SubmitsTraces(string packageVersion)
             int agentPort = TcpPortProvider.GetOpenPort();
 
             using (var agent = new MockTracerAgent(agentPort))
-            using (ProcessResult processResult = RunSampleAndWaitForExit(agent.Port, packageVersion: packageVersion))
+            using (ProcessResult processResult = RunSampleAndWaitForExit(agent.Port, packageVersion: packageVersion, enableNetStandard: true))
             {
                 Assert.True(processResult.ExitCode >= 0, $"Process exited with code {processResult.ExitCode}");
 
                 var spans = agent.WaitForSpans(expectedSpanCount, operationName: expectedOperationName);
 
-                // HACK: I get 38 spans locally but CI gets 48.
```

Comment:
Yikes, I forgot about this horrible hack. 🤦🏽

Reply:
But the hack didn't forget you 👻